lib/wallaroo/core/source/connector_source/connector_source_config.pony

```diff
@@ -24,8 +24,8 @@ use "wallaroo/core/source"
 use "wallaroo_labs/mort"
 
 primitive ConnectorSourceConfigCLIParser
-  fun apply(args: Array[String] val):
-    Map[SourceName, ConnectorSourceConfigOptions] val ?
+  fun apply(source_name: SourceName, args: Array[String] val):
```

Comment:
Would this change break other existing apps not included in this PR?  Or is this the only app using the CLIParser currently?

Reply:
This is the only one that I'm aware of.